hudi-cli/src/main/java/org/apache/hudi/cli/commands/TableCommand.java

```diff
@@ -87,7 +87,7 @@ public String createTable(
           help = "Hoodie Table Type. Must be one of : COPY_ON_WRITE or MERGE_ON_READ") final String tableTypeStr,
       @CliOption(key = {"archiveLogFolder"}, help = "Folder Name for storing archived timeline") String archiveFolder,
       @CliOption(key = {"layoutVersion"}, help = "Specific Layout Version to use") Integer layoutVersion,
-      @CliOption(key = {"payloadClass"}, unspecifiedDefaultValue = "org.apache.hudi.common.model.HoodieAvroPayload",
+      @CliOption(key = {"payloadClass"}, unspecifiedDefaultValue = "org.apache.hudi.common.model.OverwriteWithLatestAvroPayload",
```

Comment:
Let's retain the old payload class for now. We can rethink about this as part of upgrade-downgrade (https://issues.apache.org/jira/browse/HUDI-1172)